apps/ui/src/components/Ui/LinkPreview.vue

```diff
@@ -27,19 +29,25 @@ debouncedWatch(
 </script>
 
 <template>
-  <div v-if="preview?.meta?.title" class="!flex items-center border rounded-lg">
-    <div v-if="preview?.links?.icon?.[0]?.href" class="px-4 pr-0">
-      <div class="w-[32px]">
-        <img :src="preview.links.icon[0].href" width="32" height="32" class="bg-white rounded" />
+  <div v-if="preview?.meta || showDefault" class="!flex items-center border rounded-lg">
+    <template v-if="preview?.meta?.title">
+      <div v-if="preview?.links?.icon?.[0]?.href" class="px-4 pr-0">
+        <div class="w-[32px]">
+          <img :src="preview.links.icon[0].href" width="32" height="32" class="bg-white rounded" />
+        </div>
       </div>
-    </div>
-    <div class="px-4 py-3 overflow-hidden">
-      <div class="text-skin-link truncate" v-text="preview.meta.title" />
-      <div
-        v-if="preview.meta.description"
-        class="text-[17px] text-skin-text truncate"
-        v-text="preview.meta.description"
-      />
+      <div class="px-4 py-3 overflow-hidden">
+        <div class="text-skin-link truncate" v-text="preview.meta.title" />
+        <div
+          v-if="preview.meta.description"
+          class="text-[17px] text-skin-text truncate"
+          v-text="preview.meta.description"
+        />
+      </div>
+    </template>
+    <div v-else-if="showDefault" class="p-3 py-2 flex gap-2 items-center w-full">
```

Comment:
Can you use same spacing as the preview?
```suggestion
    <div v-else-if="showDefault" class="px-4 py-3 flex gap-2 items-center w-full">
```